src/reference/asciidoc/whats-new.adoc

```diff
@@ -11,3 +11,8 @@ The class `ContainerProperties` has been moved from `org.springframework.kafka.l
 The enum `AckMode` has been moved from `AbstractMessageListenerContainer` to `ContainerProperties`.
 
 `setBatchErrorHandler()` and `setErrorHandler()` methods have been moved from `ContainterProperties` to `AbstractMessageListenerContainer` (and `AbstractKafkaListenerContainerFactory`).
+
+==== After rollback processing
+
+A new `AfterRollbackProcessor` strategy is provided - see <<after-rollback>> for more information.
+(Added in 2.1.6).
```

Comment:
`1.3.5` ?

Reply:
Doh - I forgot again that master is 2.2.x.

I guess we can just remove this what's new on master but add it to 2.1.x and 2.0.x (with the right version).

Reply:
OK. Will do on merge.

The `1.3.x` back-port for you: too much Java 8 code base :smile:

Reply:
Yes, indeed; that's what I said in the commit comment.